e2e/cephfs.go

```diff
@@ -172,549 +194,541 @@ var _ = Describe("cephfs", func() {
 			appSmartClonePath := cephfsExamplePath + "pod-clone.yaml"
 			snapshotPath := cephfsExamplePath + "snapshot.yaml"
 
-			By("checking provisioner deployment is running")
-			var err error
-			err = waitForDeploymentComplete(cephfsDeploymentName, cephCSINamespace, f.ClientSet, deployTimeout)
-			if err != nil {
-				Fail(err.Error())
-			}
+			By("checking provisioner deployment is running", func() {
+				err := waitForDeploymentComplete(cephfsDeploymentName, cephCSINamespace, f.ClientSet, deployTimeout)
+				if err != nil {
+					e2elog.Failf("timeout waiting for deployment %s %v", cephfsDeploymentName, err)
+				}
+			})
 
-			By("checking nodeplugin deamonsets is running")
-			err = waitForDaemonSets(cephfsDeamonSetName, cephCSINamespace, f.ClientSet, deployTimeout)
-			if err != nil {
-				Fail(err.Error())
-			}
+			By("checking nodeplugin deamonsets is running", func() {
+				err := waitForDaemonSets(cephfsDeamonSetName, cephCSINamespace, f.ClientSet, deployTimeout)
+				if err != nil {
+					e2elog.Failf("timeout waiting for daemonset %s %v", cephfsDeamonSetName, err)
+				}
+			})
 
 			By("check static PVC", func() {
 				scPath := cephfsExamplePath + "secret.yaml"
 				err := validateCephFsStaticPV(f, appPath, scPath)
```

Comment:
CephFS instead of CephFs

Reply:
not modified in this PR, we can have a follow-up PR for changing cephfs to CephFS